tests/test_api.py

```diff
@@ -45,18 +45,20 @@ def setUpClass(cls):
     def setUp(self):
         # pylint: disable=consider-using-with
         self.mock_log = tempfile.NamedTemporaryFile()
-        self.p_mock = self.spawn_server('org.freedesktop.Test',
-                                        '/',
-                                        'org.freedesktop.Test.Main',
-                                        stdout=self.mock_log)
+        self.p_mock = self.spawn_server("org.freedesktop.Test", "/", "org.freedesktop.Test.Main", stdout=self.mock_log)
 
-        self.obj_test = self.dbus_con.get_object('org.freedesktop.Test', '/')
-        self.dbus_test = dbus.Interface(self.obj_test, 'org.freedesktop.Test.Main')
+        self.obj_test = self.dbus_con.get_object("org.freedesktop.Test", "/")
+        self.dbus_test = dbus.Interface(self.obj_test, "org.freedesktop.Test.Main")
         self.dbus_mock = dbus.Interface(self.obj_test, dbusmock.MOCK_IFACE)
         self.dbus_props = dbus.Interface(self.obj_test, dbus.PROPERTIES_IFACE)
 
     def assertLog(self, regex):
-        self.assertRegex(Path(self.mock_log.name,).read_bytes(), regex)
+        self.assertRegex(
+            Path(
+                self.mock_log.name,
+            ).read_bytes(),
+            regex,
+        )
```

Comment:
Ugh -- this is "I get paid by written lines of code" vandalism. Why is black so trigger-happy with "put a single word on each line"? This feels quite arbitrary, especially as [further below](https://github.com/martinpitt/python-dbusmock/pull/188/files#diff-3451cc474d5698222168f66a293339a6bd296ee2c56f56409866bbae99917a79L255) it did the exact opposite :thinking: 

After some interactive experimenting: the whole problem is that stray `,` in the Path() call. If you drop this, then black leaves it alone.